cluster/addons/fluentd-elasticsearch/fluentd-es-image/td-agent.conf

```diff
@@ -40,10 +40,16 @@
   path /varlog/containers/*.log
   pos_file /varlog/es-containers.log.pos
   time_format %Y-%m-%dT%H:%M:%S
-  tag kubernetes.*
+  tag reform.*
   read_from_head true
 </source>
 
+<match reform.**>
+  type record_reformer
+  enable_ruby true
+  tag kubernetes.${tag_suffix[3].split('-')[0..-2].join('-')}
```

Comment:
A comment showing what the tag is before and after this operates on it would be very helpful to future maintainers. Can you please add that in your followup PR?